src/Statics/Kses.php

```diff
@@ -25,9 +27,14 @@ class Kses {
 	 *
 	 * @return void
 	 *
+	 * @throws GravityPdfException
 	 * @since 6.4.0
 	 */
-	public static function output( string $html ): void {
+	public static function output( $html ): void {
+		/* If the $html is not a valid string, throw an exception if this returns false. */
+		if ( ! is_string( $html ) ) {
```

Comment:
@jestonihpi I think moving this into self::parse() would be better. It would cover any uses of the Kses class that way.

Reply:
Copy @jakejackson1.